collectd-frontend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:5.6-apache
```

Comment:
The repo readme mentions both, that's why I asked. If Apache works better here — fair enough.